specviz/plugins/model_editor/model_editor.py

```diff
@@ -259,7 +259,10 @@ def _add_fittable_model(self, model_type):
         spec = self._get_selected_plot_data_item().data_item.spectrum
 
         if inc_regs is not None:
-            spec = extract_region(spec, inc_regs)
+            exc_regs = inc_regs.invert_from_spectrum(spec) \
+                if inc_regs is not None else None
```

Comment:
For consistency this seems to be the same structure as above change, so how about just:
```
       exc_regs = inc_regs.invert_from_spectrum(spec) \
            if inc_regs is not None else None
```
though maybe I am missing something.

Reply:
nvm, missed the line below.